### PR TITLE
Docs 1.0 refactor installation

### PR DESCRIPTION
**Pull request by markurtz**

### CODE REVIEW

File: src/content/get-started/install.mdx [hunk 1]
```diff
@@ -1,9 +1,90 @@
 ---
 title: "Install"
-metaTitle: "Install"
-metaDescription: "Install"
+metaTitle: "Install DeepSparse Platform"
```
Comment (jeanniefinks) [inline src/content/get-started/install.mdx:3]:
```suggestion
metaTitle: "Install Deep Sparse Platform"
```

File: src/content/get-started/install.mdx [hunk 2]
```diff
@@ -1,9 +1,90 @@
...
+metaDescription: "Install instructions for the DeepSparse Platform including DeepSparse, SparseML, SparseZoo"
```
Comment (jeanniefinks) [inline src/content/get-started/install.mdx:4]:
```suggestion
metaDescription: "Installation instructions for the Deep Sparse Platform including DeepSparse, SparseML, SparseZoo"
```

**Review by jeanniefinks — APPROVED** [review]

File: src/content/get-started/install/deepsparse.mdx [hunk 3]
```diff
@@ -0,0 +1,41 @@
+---
+title: "DeepSparse Engine"
+metaTitle: "DeepSparse Engine Installation"
+metaDescription: "Installation instructions for the DeepSparse Inference Engine for neural network deployments."
```
Comment (jeanniefinks) [inline src/content/get-started/install/deepsparse.mdx:4]:
```suggestion
metaDescription: "Installation instructions for the DeepSparse Engine for neural network deployments"
```

File: src/content/get-started/install/sparsezoo.mdx [hunk 4]
```diff
@@ -0,0 +1,23 @@
+---
+title: "SparseZoo"
+metaTitle: "SparseZoo Installation"
+metaDescription: "Installation instructions for the SparseZoo sparse model repository."
```
Comment (jeanniefinks) [inline src/content/get-started/install/sparsezoo.mdx:4]:
```suggestion
metaDescription: "Installation instructions for the SparseZoo sparse model repository"
```

**Review by jeanniefinks — CHANGES_REQUESTED** [review]
Minor edits

**Comment by markurtz** [comment]
@jeanniefinks edits merged in

**Pull request merged**